web/src/components/tasks/cigarette-license/LicenseeInfo.test.tsx

```diff
@@ -0,0 +1,211 @@
+import { LicenseeInfo } from "@/components/tasks/cigarette-license/LicenseeInfo";
+import { AddressContext } from "@/contexts/addressContext";
+import { CigaretteLicenseContext } from "@/contexts/cigaretteLicenseContext";
+import { getMergedConfig } from "@/contexts/configContext";
+import {
+  createDataFormErrorMap,
+  DataFormErrorMapContext,
+} from "@/contexts/dataFormErrorMapContext";
+import { ProfileDataContext } from "@/contexts/profileDataContext";
+import { useMockRoadmap } from "@/test/mock/mockUseRoadmap";
+
+import { WithStatefulUserData } from "@/test/mock/withStatefulUserData";
+import { emptyCigaretteLicenseData } from "@businessnjgovnavigator/shared/cigaretteLicense";
+import { emptyFormationAddressData } from "@businessnjgovnavigator/shared/formationData";
+import {
+  generateBusiness,
+  generateFormationData,
+  generateFormationFormData,
+  generateProfileData,
+  generateUserDataForBusiness,
+} from "@businessnjgovnavigator/shared/test";
+import { Business, UserData } from "@businessnjgovnavigator/shared/userData";
+import { render, screen, within } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import { useState } from "react";
+
+const Config = getMergedConfig();
+
+jest.mock("@/lib/data-hooks/useRoadmap", () => ({ useRoadmap: jest.fn() }));
+
+describe("<LicenseeInfo />", () => {
+  beforeEach(() => {
+    jest.resetAllMocks();
+    useMockRoadmap({});
+  });
+
+  const renderComponent = ({
+    business,
+    userData,
+    setStepIndex = jest.fn(),
+    runValidations = true,
+  }: {
+    business?: Business;
+    userData?: UserData;
+    setStepIndex?: (idx: number) => void;
+    runValidations?: boolean;
+  } = {}): void => {
+    const testBusiness = business ?? generateBusinessWithDefaults();
+    const testUserData = userData ?? generateUserDataForBusiness(testBusiness);
+
+    const TestComponent = (): JSX.Element => {
+      const [cigaretteLicenseData, setCigaretteLicenseData] = useState(emptyCigaretteLicenseData);
+      const [profileData, setProfileData] = useState(testBusiness.profileData);
+
+      return (
+        <WithStatefulUserData initialUserData={testUserData}>
+          <DataFormErrorMapContext.Provider
+            value={{
+              fieldStates: createDataFormErrorMap(),
+              runValidations,
+              reducer: () => {},
+            }}
+          >
+            <AddressContext.Provider
+              value={{
+                state: {
+                  formationAddressData: emptyFormationAddressData,
+                },
+                setAddressData: jest.fn(),
```

Comment:
Decided to leave this as is - the licensee info component does expect AddressContext but we're not testing any real Address related behavior here